trecs/tests/test_Users.py

```diff
@@ -115,3 +115,21 @@ def test_attention(self):
         users.compute_user_scores(items)
         feedback = users.get_user_feedback(items_shown=items_shown)
         np.testing.assert_array_equal(feedback, np.ones(num_users))
+
+    def test_no_repeated_items(self):
+        num_users = 5
+        num_attr = 4
+        num_items = 5
+        users = Users(np.ones((num_users, num_attr)), repeat_interactions=False)
+        # first item is most desirable because its attributes are all 0,
+        # second item is second most desirable because attributes are all -1, etc.
+        items = np.tile(np.arange(num_items) * -1, (num_attr, 1))
+        users.compute_user_scores(items)
+        items_shown = np.tile(np.arange(num_items), (num_users, 1))  # all items shown to all users
+        # all users should like the 0th item best
+        feedback = users.get_user_feedback(items_shown=items_shown)
```

Comment:
I think this is going to fail on DN users or other instances where user behavior is non-deterministic, but we can kick that can down the road

Reply:
yes, fair enough! I think this test class is only targeted for the default `Users` behavior but we should also write more tests for `DNUsers` as well.